xclient_gui/desktop/views/components/chat.py

```diff
@@ -1,48 +1,38 @@
-from PyQt5.QtWidgets import QVBoxLayout, QWidget, QFrame
+from PyQt5.QtWidgets import QListWidget, QWidget, QVBoxLayout, QLineEdit, QHBoxLayout, QPushButton, \
+    QListWidgetItem
 
-from xclient_gui.desktop.views.components.tree import Tree
 
-
-class Chat(QWidget):
+class MessagesList(QListWidget):
     def __init__(self):
         super().__init__()
+        self.setStyleSheet('background-color:gray')
+        self.setWordWrap(True)
+        self.show()
 
-        # self.addItem(QLayoutItem())
-        # self.addItem("Item 2")
-        # self.addItem("Item 3")
-        # self.addItem("Item 4")
-        # self.addItem("Item 5")
-        # self.setView(QListView())
-        # box = QVBoxLayout
-
-        #
-
-        hbox = QVBoxLayout()
-
-        tree = Tree()
-        tree.setFrameShape(QFrame.StyledPanel)
-        tree.setStyleSheet('background-color:white')
 
-        tree2 = Tree()
-        tree2.setFrameShape(QFrame.StyledPanel)
-        tree2.setStyleSheet('background-color:red')
+class NewMessageInput(QHBoxLayout):
+    def __init__(self):
+        super().__init__()
 
-        # chat = Chat()
-        # chat.setStyleSheet('background-color:green')
+        message_input = QLineEdit()
+        button = QPushButton(">")
+        self.addWidget(message_input)
+        self.addWidget(button)
 
-        # splitter = QSplitter(Qt.Vertival)
-        # splitter.addWidget(tree)
-        # splitter.addWidget(tree2)
 
-        # splitter.setSizes([50, 200])
+class Chat(QWidget):
+    def __init__(self):
+        super().__init__()
 
-        # hbox.direction = QVBoxLayout.BottomToTop
+        layout = QVBoxLayout()
 
-        hbox.addWidget(tree)
-        hbox.addWidget(tree2)
+        messages = MessagesList()
 
-        self.setLayout(hbox)
-        self.show()
+        messages.addItem("Witaj w pokoju Test1.\nLista aktywnych użytkowników: Jan Kowalski, Pan Zbysiu, Zenek")
```

Comment:
Text in English, please

Reply:
This text is only temporary, to test if everything is displaying right and will be removed in the next pull request.